CDTS Blazor/Data/Services/AzureKeyVaultService.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace CDNApplication.Data.Services
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.KeyVault;
+    using Microsoft.Azure.KeyVault.Models;
+    using Microsoft.Azure.Services.AppAuthentication;
+    using Microsoft.Rest.Azure;
+
+    /// <summary>
+    ///  Represents the Azure key vault service.
+    /// </summary>
+    public class AzureKeyVaultService
+    {
+        private string dNs;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureKeyVaultService"/> class.
+        /// </summary>
+        /// <param name="dNS">DNS for the key vault.</param>
+        public AzureKeyVaultService(string dNS)
+        {
+            this.dNs = dNS;
+        }
+
+        /// <summary>
+        /// Gets a secret from azure key vault.
+        /// </summary>
+        /// <param name="secretName">Secret to get.</param>
+        /// <returns>Secret value.</returns>
+        public string GetSecret(string secretName)
```

Comment:
Suggest renaming the GetSecret method to be more clear about the intentions of the method.
```suggestion
        public string GetSecretByName(string secretName)
```